.github/workflows/test-connect.yml

```diff
@@ -128,41 +123,23 @@ jobs:
       fail-fast: false
       matrix: ${{ fromJson(needs.set-matrix.outputs.dailyMatrix) }}
 
-  legacy-fw:
-    needs: [build, set-matrix]
-    name: legacy-fw ${{ matrix.key }}
-    if: (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch') && github.repository == 'trezor/trezor-suite'
-    uses: ./.github/workflows/template-connect-test-params.yml
-    with:
-      testPattern: ${{ matrix.groups.pattern }}
-      includeFilter: ${{ matrix.groups.includeFilter }}
-      testsFirmware: ${{ matrix.firmware }}
-      testDescription: ${{ matrix.groups.pattern }}-${{ matrix.groups.name }}-${{ matrix.env }}
-      cache_tx: ${{ matrix.cache_tx }}
-      transport: ${{ matrix.transport }}
-      testEnv: ${{ matrix.env }}
-      testFirmwareModel: ${{ matrix.model }}
-    strategy:
-      fail-fast: false
-      matrix: ${{ fromJson(needs.set-matrix.outputs.legacyFirmwareMatrix) }}
-
-  canary-fw:
+  all-fws:
     needs: [build, set-matrix]
-    name: canary-fw ${{ matrix.key }}
+    name: all-fws ${{ matrix.key }}
```

Comment:
Maybe we could put `${{ matrix.firmware }}` here so it's grouped

Reply:
something like this? https://github.com/trezor/trezor-suite/actions/runs/11780696955